product_docs/docs/pgbouncer/1.0/index.mdx

```diff
@@ -13,13 +13,24 @@ legacyRedirectsGenerated:
 
 
 
-When a client application connects to a Postgres server, it negotiates a connection; that negotiation takes time. PgBouncer saves time by maintaining a pool of pre-established connections to the server. Instead of connecting directly to the server, the client connects to PgBouncer, minimizing the connection negotiation time by using a previously established connection made available to the client in the PgBouncer pool.
+When a client application connects to a Postgres server, it negotiates a connection; that negotiation takes time. PgBouncer saves time by maintaining a pool of pre-established connections to the server. Instead of connecting directly to the server, the client connects to PgBouncer, minimizing the connection negotiation time using a previously established connection made available to the client in the PgBouncer pool.
 
-PgBouncer is a lightweight connection pooling utility for Postgres and Advanced Server installations that is based on the openSource PgBouncer project.
+PgBouncer is a lightweight connection pooling utility for Postgres and Advanced Server installations, based on the openSource PgBouncer project.
 
-EnterpriseDB enhancements for the PgBouncer project are available via RPM Packages, deb packages, or a graphical installer that you can download with StackBuilder Plus. The enhancements allow PgBouncer to service clients that are using EDB Connectors that require compatible out parameter handling.
+EnterpriseDB enhancements for the PgBouncer project are available via RPM Packages, deb packages, or a graphical installer that you can download with StackBuilder Plus. These enhancements allow PgBouncer to service clients that are using EDB Connectors that require compatible out parameter handling.
 
 For more information about PgBouncer, including reference and usage information, please visit the [PgBouncer project site](https://www.pgbouncer.org).
 
 !!! Note
-    If you are using the pdf version of this document, using cut/paste to copy command may result in extra spaces or carriage returns in the pasted command. If a command fails, check the command carefully for additional characters.
+
+    </div>
```

Comment:
these orphan tags need to be removed. This won't render correctly.

Reply:
Removed, please check